cfme/infrastructure/provider/rhevm.py

```diff
@@ -96,3 +96,8 @@ def from_config(cls, prov_config, prov_key, appliance=None):
                    start_ip=start_ip,
                    end_ip=end_ip,
                    appliance=appliance)
+
+    @staticmethod
+    def discover_dict():
+        """Returns the discovery dict"""
+        return {"rhevm": True}
```

Comment:
This might as well be just a class attribute, I see no reason why this should be hidden in a function call :)